### DISCUSSION TITLE
Error while migration "cannot debit"

### DISCUSSION THREAD

**Discussion by theregenfarmer** [iotaledger/legacy-migration-tool#142] [Help & Support]
IOTA Balance is correctly shown, but it shows 0 $. Therefore I guess that error accurs....Whats the solution?

Tue, 03 Dec 2024 11:52:49 GMT
migration error: cannot debit (base tokens: 666157497) from 05204969@iota1pphx6hnmxqdqd2u4m59e7nvmcyulm3lfm58yex5gmud9qlt3v9crs9sah6m: not enough funds

**Comment by vbgweb** [comment]
Hi, same here. Waiting for possible solutions

**Comment by JuanJoseNacher** [comment]
Hola buenas, yo también tengo el mismo problema el saldo se muestra correctamente. Creo que será un error interno al hacer la migración. A ver si nos puede ayudar la fundación de IOTA.  En mi caso me ocurre lo mismo y no puedo hacer la migracion a la nueva wallet.
errorMessage: cannot debit (base tokens: 25064191353) from 05204969@iota1pphx6hnmxqdqd2u4m59e7nvmcyulm3lfm58yex5gmud9qlt3v9crs9sah6m: not enough funds

**Comment by EURODRY** [comment]
Hello everyone. I have the same problem... insufficient funds for transfer. Maybe someone can help us too.

**Comment by amin3z777** [comment]
I have same problem.
I have 575 IOTA but shows , not enough funds ,
what should we do?

**Comment by begonaalvarezd** [comment]
The issue `errorMessage: cannot debit (base tokens: XXX) from YYY: not enough funds` has been mitigated, please try the migration again. We apologize for the troubles :pray:
